src/Tribe/Tickets_View.php

```diff
@@ -196,7 +197,13 @@ public function update_tickets() {
 
 		$post_id = get_the_ID();
 
-		$attendees = ! empty( $_POST['attendee'] ) ? $_POST['attendee'] : [];
+		$attendees = [];
+
+		if ( isset( $_POST['tribe_ticket_attendee'] ) ) {
+			$attendees = $_POST['tribe_ticket_attendee'];
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['tribe_ticket_attendee']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: d17fdc5a30f5f90d3da73d64ab717ecc</sub>